src/client/Microsoft.Identity.Client.Broker/BrokerExtension.cs

```diff
@@ -80,12 +80,21 @@ private static void AddRuntimeSupport(PublicClientApplicationBuilder builder)
                          return new RuntimeBroker(uiParent, appConfig, logger);
                      };
             }
+            else if (DesktopOsHelper.IsMac())
+            {
+                 builder.Config.BrokerCreatorFunc =
+                     (uiParent, appConfig, logger) =>
+                     {
+                         logger.Info("[Runtime] macOS supports broker.");
+                         return new RuntimeBroker(uiParent, appConfig, logger);
+                     };
+            }
             else
             {
                 builder.Config.BrokerCreatorFunc =
                    (uiParent, appConfig, logger) =>
                    {
-                       logger.Info("[RuntimeBroker] Not a Windows 10 or Server equivalent machine. Runtime broker or SsoPolicy support is not available.");
+                       logger.Info("[RuntimeBroker] not available in current platform.");
```

Comment:
can you please add the current os type? 

```suggestion
                       logger.Warning($"[RuntimeBroker] Not available on the current platform: {osType}.");
```